app/deps/sdl.sh

```diff
@@ -35,45 +35,48 @@ else
     cd "$DIRNAME"
 
     conf=(
-        --prefix="$INSTALL_DIR/$DIRNAME"
+        -DCMAKE_INSTALL_PREFIX="$INSTALL_DIR/$DIRNAME"
     )
 
     if [[ "$HOST" == linux ]]
     then
         conf+=(
-            --enable-video-wayland
-            --enable-video-x11
+            -DSDL_WAYLOAD=ON
```

Comment:
You probably mean SDL_WAYLAND?

Reply:
Good catch :+1:

Indeed, in the logs:

```
 CMake Warning:
-- Generating done (0.1s)
  Manually-specified variables were not used by the project:

    SDL_WAYLOAD
```

(it should be an error instead of a warning)